tools/pipelines/templates/build-npm-package.yml

```diff
@@ -407,6 +412,37 @@ extends:
                         inputs:
                           sourceFolder: '${{ parameters.buildDirectory }}/nyc/report'
                           targetFolder: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
+                      - task: Bash@3
+                        displayName: Report Code Coverage Comparison
+                        condition: and(succeededOrFailed(), eq('${{ parameters.reportCodeCoverageComparison }}', true), eq(variables['ReportDirExists'], 'true'), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['System.PullRequest.TargetBranch'], 'main'))
+                        continueOnError: false
+                        env:
+                          ADO_API_TOKEN: $(System.AccessToken)
+                          GITHUB_API_TOKEN: $(githubPublicRepoSecret)
+                          TARGET_BRANCH_NAME: '$(targetBranchName)'
+                          ADO_BUILD_ID: $(Build.BuildId)
+                          GITHUB_PR_NUMBER: $(System.PullRequest.PullRequestNumber)
+                          ADO_CI_BUILD_DEFINITION_ID_BASELINE: 48
+                          ADO_CI_BUILD_DEFINITION_ID_PR: 11
+                        inputs:
+                          targetType: inline
+                          workingDirectory: '${{ parameters.buildDirectory }}'
+                          script: |
+                            set -eux -o pipefail
+                            REPO_NAME=$(echo $BUILD_REPOSITORY_NAME | cut -d'/' -f2)
+                            REPO_OWNER=$(echo $BUILD_REPOSITORY_NAME | cut -d'/' -f1)
```

Comment:
No using bash!

Reply:
Instead of splitting this value here, have the command accept a single flag with the combined value and split in the command if you need to.

Reply:
What is the issue with bash and if there is some issue, I should be able to use something else?
I still want to split it here rather than in the tool, because both are worked with separately in the git APIs, rather than together. So, I prefer to split here.

Reply:
Is it about the combability with windows?

Reply:
.

Reply:
Removed it